### DISCUSSION TITLE
SCTP vs MTCP

### DISCUSSION THREAD

**Discussion by ghost** [dtn7/dtn7-rs#33] [Q&A] [answered]
Why was MTCP used as a convergence layer over SCTP (Stream Transmission Control Protocol), what benefits does MTCP offer?

**Comment by gh0st42** [comment, marked as answer]
Simply because MTCP was the only protocol with a draft designed for the then new bundle protocol v7, the regular TCP CL was not even in draft state when we started implementing dtn7. 
Our `dtnd` has support for other convergence layers as well, and you can choose a different one, e.g., `tcp` or `http`, or implement some yourself if you need different ones. 
We are always open for contributions :)
